buildpacks/java/java/build.go

```diff
@@ -110,3 +113,21 @@ func findPath(path string, r *regexp.Regexp) (string, error) {
 	}
 	return "", nil
 }
+
+func (b Build) convertLabels(t interface{}) []libcnb.Label {
+	sliceOfMaps := t.([]map[string]interface{})
+	labels := []libcnb.Label{}
+	for _, mapOfLabels := range sliceOfMaps {
+		pairs := []string{}
+		for _, val := range mapOfLabels {
+			pairs = append(pairs, val.(string))
+		}
+		labels = append(labels,
+			libcnb.Label{
+				Key:   pairs[0],
+				Value: pairs[1],
+			})
+		pairs = nil
```

Comment:
Don't think this is needed, since you're using `pairs := []string{}` on line 121